src/main/java/ca/team2706/frc/robot/Sendables.java

```diff
@@ -35,7 +35,11 @@ public static SendableBase newPigeonSendable(PigeonIMU pigeonIMU) {
         return new SendableBase() {
             @Override
             public void initSendable(SendableBuilder builder) {
-                builder.addDoubleProperty(PIGEON_NAME, pigeonIMU::getFusedHeading, pigeonIMU::setFusedHeading);
+                builder.addDoubleProperty(PIGEON_NAME, () -> {
+                    double[] yawPitchRoll = new double[3];
+                    pigeonIMU.getYawPitchRoll(yawPitchRoll);
```

Comment:
I think it would make sense to add a method specifically for getting the robot's yaw, pitch and roll, since doing the whole thing with the double array each time is tedious.